charts/intel-resource-drivers/README.md

```diff
@@ -1,4 +1,4 @@
-# Dynamic Resource Allocation (DRA) Intel GPU Driver Helm Chart
+# Dynamic Resource Allocation (DRA) Intel Drivers Helm Chart
```

Comment:
Please add documentation about contents of this chart, what it includes, perhaps link to the resource-drivers' GitHub repo